src/components/plugins/PluginIndex.vue

```diff
@@ -82,6 +82,35 @@
     }
 
     const elementsByType = computed<Record<string, string[]>>(() => extractPluginElements(plugin.value));
+    const groupedByAction = computed(() => {
+        const result: Record<string, object> = {};
+
+        const allTasks = Object.keys(elementsByType.value);
+
+        allTasks.forEach((task) => {
```

Comment:
In javascript, one can loop on objects of they are itterable with the [statement `for(... in ...)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)

```suggestion
        for(const task in elementsByType)
```

using this instead of functional `forEach()` allows more flexibility and less parenthesis clutter.
It is also often faster.